word-guess

```diff
@@ -0,0 +1,171 @@
+require 'colorize'
+class Picture
+  attr_accessor :parachute_man, :life
+  def initialize
+    @parachute_strings = ['  \\ ' ,'  |','_','|  ', ' /']
+    @life = 4
+    top
+    print_strings
+    puts
+    bottom
+
+  end
+  def top
+    puts "   _________".colorize(:blue)
+    puts '  /         \ '.colorize(:blue)
+    puts ' / _   _   _ \ '.colorize(:blue)
+    puts ' |/ \\ / \\ / \\|'.colorize(:blue)
+  end
+
+  def bottom
+    puts  '   o  (_} o '.colorize(:white)
+    puts  '    \\/.X.\\/'.colorize(:white)
+    puts  '      |_|'.colorize(:white)
+    puts  '     // \\\\ '.colorize(:white)
+    puts  '     \\\\ //'.colorize(:white)
+    puts  '      U U'.colorize(:white)
+  end
+  def print_strings
+    @parachute_strings.each do |string|
+      print string.colorize(:magenta)
+    end
+  end
+
+  # def removing_strings
+  #   case
+  #   when @life == 4
+  #     print_strings
+  #   when @life == 3
+  #     @parachute_strings[-1] = " "
+  #     print_strings
+  #   when @life == 2
+  #     @parachute_strings[-1] = " "
+  #     @parachute_strings[-2] = " "
+  #     print_strings
+  #   when @life == 1
+  #     @parachute_strings[-1] = " "
+  #     @parachute_strings[-2] = " "
+  #     @parachute_strings[-4] = " "
+  #     print_strings
+  #   when @life == 0
+  #     @parachute_strings[-1] = " "
+  #     @parachute_strings[-2] = " "
+  #     @parachute_strings[-4] = " "
+  #     @parachute_strings[0] = " "
+  #     print_strings
+  def remove1
+    @parachute_strings[-1] = " "
+    print_strings
+  end
+  def remove2
+    @parachute_strings[-1] = " "
```

Comment:
You're right that this code is duplicated, but it's also a tricky thing to figure out how to print only what you want. What if you modified the `print_strings` method to take a number of lives remaining - something like this:

```ruby
def print_strings(lives_left)
  # strings before the center strap
  (0...2).each do |i|
    if i <= lives_left
      print @parachute_strings[i].colorize(:magenta)
    else
      print " " * @parachute_strings[i].length
    end
  end

  # center strap
  print @parachute_strings[2]

  # strings after the strap
  (3...5).each do |i|
    if i <= lives_left
      print @parachute_strings[i].colorize(:magenta)
    else
      print " " * @parachute_strings[i].length
    end
  end
end
```

The center strap makes the whole thing much more complex - maybe it would be best to just take it out.